README.md

```diff
@@ -437,6 +438,31 @@ func main() {
 }
 ```
 
+<a id="markdown-using-as-a-library" name="using-as-a-library"></a>
+## Using As A Library
+
+While the source projects like
+[transport](https://github.com/asecurityteam/transport) and
+[httpstats](https://github.com/asecurityteam/httpstats) can be used directly, there
+is a great deal of convenience in being able to configure all of the combined tooling
+through the OpenAPI extensions. This is particularly true if you are moving a system
```

Comment:
Question: Just for clarification, in a more complicated example where you might have multiple different backends, routes and `x-transportd` extensions defined, would you create one "smart client" and embed it in any clients you create?

Reply:
Yeah, you'd end up using one transport from the library in _all_ clients. Internally, the smart client manages a number of different `http.Transport` instances and will route to them based on the URL path. So you still get all of the benefits we have in the past from using multiple instances of `http.Transport` for different dependencies but you don't have to perform any of the orchestration yourself to install them in the correct locations. Just use the one client everywhere.

Reply:
😎 Very cool! Might be worth putting some or all of that into the description here.